docs/_posts/2015-07-03-react-v0.14-beta-1.md

```diff
@@ -66,7 +68,7 @@ var Zoo = React.createClass({
 });
 ```
 
-This change also applies to the return result of `ReactDOM.render` when passing a DOM node as the top component. References to custom component classes work exactly as before.
+This change also applies to the return result of `ReactDOM.render` when passing a DOM node as the top component.
```

Comment:
Maybe also put some text here to drive the point home?

Some thing like: This change also applies to the return result of `ReactDOM.render` when passing a DOM node (eg. `span`, `div`, `img`) as the top component.  As with refs, this change does not affect custom components (eg. `<MyFancyMenu>` or `<MyContextProvider>`), which remain unaffected by this change.